src/main/java/me/coley/recaf/util/ClassUtil.java

```diff
@@ -22,9 +22,14 @@
 	 */
 	public static ClassReader fromRuntime(String name) {
 		try {
-			Class<?> loaded = ClasspathUtil.getSystemClass(normalize(name));
-			return new ClassReader(loaded.getName());
-		} catch(ClassNotFoundException | IOException e) {
+			URL url = ClassLoader.getSystemResource(name.replace('.', '/').concat(".class"));
```

Comment:
I was under the assumption that using `concat` for optimization was no longer effective - https://stackoverflow.com/a/46485284